src/patterns/ec2-app.ts

```diff
@@ -48,6 +50,12 @@ interface RestrictedAccess extends Access {
 
 export type AppAccess = PublicAccess | RestrictedAccess;
 
+export interface AccessLoggingProps {
+  // This provides a hint & incentive to users to use a recommended path
+  bucketSSMPath: string | "/account/services/access-logging/bucket";
```

Comment:
Since `GuSSMParameter` doesn't create a CFN parameter in the UI (and therefore means we can't set sensible defaults, what you write is what you get), this means we have a handy way of hinting that this is the preferred SSM path to use without enforcing it.

Reply:
Hmm, any reason to not enforce it?

Reply:
I guess not, but if we enforce it, we risk users having less visibility on where the construct will look for the value in SSM. However, as long as we document it within the pattern, I suppose this could be okay? Users would either a) not have to worry about it anyways since the SSM parameter is created in the account, or b) click through to the prop definition, where this path is dictated.

Reply:
I would probably vote for enforcing this and adding it to the docs.

Reply:
👍🏽  to enforcing it. It'll be easier to make it flexible later once we have a use case, rather than restricting it later.

Reply:
Yeah, good call both. Thinking about it, I agree that enforcing it makes sense. I've just pushed a change that enforces it as well as makes the prefix mandatory, as you'd otherwise have the following if you wanted to omit the prefix:

```typescript
{
// other props
  accessLogging: {}
}
```

Now, it looks like this:
```typescript
{
// other props
  accessLogging: { prefix: "some-prefix" }
}

{
// other props
  accessLogging: { prefix: false }
}
```

Would be keen to hear your thoughts on whether this is nice enough.